chrome/run.js

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+const chromeLaunch = require('chrome-launch'); // eslint-disable-line import/no-extraneous-dependencies
+const url = 'https://webcompat.com/'; // this can be any url, we can also pass through the command line
+const args = ['--load-extension=./dist/chrome'];
```

Comment:
nit: can we get a new line between the variable declarations and calling `chromeLaunch`?